apps/validate/validate.py

```diff
@@ -90,10 +92,11 @@ def _validate_type_text(self, field, value):
 
     def _validate_mandatory_in_list(self, mandatory, field, value):
         """Validates if all elements from mandatory are presented in the list"""
-        for key in mandatory:
-            for key_field in mandatory[key]:
-                if not check_json(value, key, mandatory[key][key_field]):
-                    self._error(key_field, REQUIRED_FIELD)
+        if field != 'subject': # ignore in case of subject and category
```

Comment:
I think subject can be validated too.

Reply:
This is how mandatory_in_list looks for subject schema `{'scheme': {'category': 'category', 'subject': 'subject_custom'}}`. So, if either category or subject is not in the template then it gives the error. Both must be present together.

Reply:
I think this code is covered by the tests in ValidateMandatoryInListTest
Looks like field is ignored because only subject can have mandatory_in_list so I think field != 'subject' is always false.
What value did you receive here?

Reply:
yes, `field != 'subject'` is always false.

Reply:
What value did you receive here?

Reply:
you mean for this `def _validate_mandatory_in_list(self, mandatory, field, value):` ?
mandatory = `{'scheme': {'category': 'category', 'subject': 'subject_custom'}}`
field = `subject` and
value = [{'name': 'Arkeologi', 'qcode': '01001000', 'parent': '01000000', 'scheme': 'subject_custom'}]

Reply:
OK, so category field is missing and you will get a validation error for 'category'.

Reply:
yes, but shouldn't it be like if I select a value for `subject` then `category` field must be visible in the editor? Because I can see only `ANPA category` but not `category`.

Reply:
yes, exactly